src/clr-angular/wizard/wizard-page.spec.ts

```diff
@@ -483,6 +483,24 @@ export default function(): void {
         });
       });
 
+      describe('hasError', () => {
+        it('should be false by default', () => {
+          expect(testWizardPage.hasError).toBe(false);
+        });
+
+        it('should be true if hasError is true', () => {
+          testWizardPage.hasError = true;
+          fixture.detectChanges();
+          expect(testWizardPage.hasError).toBe(true);
+        });
+
+        it('should return false if hasError is false', () => {
+          testWizardPage.hasError = false;
```

Comment:
You are setting the value and testing it right afterward, why?

Reply:
This PR is for #2203. The implementation was merged a few days ago and now I've done some tests for it. 

First, I made the tests after the `completed` template, and there was a test like this, but with two values. When I start writing, I made the test like there, but after that, I removed another variable and the tests remained as they are now. 
If you suggest, I'll remove or update those two tests.

Reply:
Yeah, it's a bit strange now when the other variable is gone.
Please remove them if there are no side effects that need to be tested, events fired, etc.